src/ebonite/core/objects/core.py

```diff
@@ -582,11 +590,65 @@ def __init__(self, name: str, id: str = None,
         self.params = params or {}
 
     @property
+    @_with_meta
     def model(self) -> Model:
-        raise AttributeError('Can\'t access model of unbound image')
+        m = self._meta.get_model_by_id(self.model_id)
+        if m is None:
+            raise errors.NonExistingModelError(self.model_id)
+        return m
 
     @model.setter
     def model(self, model: Model):
         if not isinstance(model, Model):
             raise ValueError('{} is not Model'.format(model))
         self.model_id = model.id
+
+
+class RuntimeEnvironment(EboniteObject):
```

Comment:
Not sure if host and port is enough for all types of environments. Maybe we need some auth? I suggest the same way as for other objects: param field

Reply:
I propose to change it after #51 because of expected merging problems